pkg/kubelet/certificate/bootstrap/bootstrap_test.go

```diff
@@ -34,9 +38,320 @@ import (
 	certificatesclient "k8s.io/client-go/kubernetes/typed/certificates/v1beta1"
 	restclient "k8s.io/client-go/rest"
 	clienttesting "k8s.io/client-go/testing"
+	"k8s.io/client-go/util/certificate"
 	"k8s.io/client-go/util/keyutil"
 )
 
+//create storeCertData of the store which used by bootstrap process. used for TestLoadClientConfig
+var storeCertData = newCertificateData(`-----BEGIN CERTIFICATE-----
```

Comment:
please put the fixture data in a standalone file in testdata, so it can be regenerated as needed

Reply:
OK! thanks, and after re-understanding, I found that this test case only needs the catalog to be available, and does not need to have specific content, which has been changed. Other test data has been placed in the designated directory. Please review again.